devito/ir/iet/utils.py

```diff
@@ -143,3 +146,38 @@ def derive_parameters(nodes, drop_locals=False):
         parameters = [p for p in parameters if not isinstance(p, LocalObject)]
 
     return parameters
+
+
+def find_offloadable_trees(iet):
```

Comment:
here's a comment which is practically useless, but conceptually important.

I'd now call this ``find_affine_trees``, as within this subdirectory (ir/iet/utils) there's conceptually no concept of "offloadable" (there's no concept of backend here)